src/main/java/com/sourcegraph/cody/agent/CodyAgentClient.java

```diff
@@ -47,24 +152,24 @@ private <T> CompletableFuture<T> onEventThread(Supplier<T> handler) {
     return result;
   }
 
+  // Webviews
```

Comment:
Curious where this comes from, seems unrelated to edits.

Reply:
That is old chat API which was removed since.